bundleresolver/cmd/bundleresolver/main.go

```diff
@@ -31,8 +33,9 @@ import (
 const timeoutDuration = time.Minute
 
 func main() {
-	sharedmain.Main("controller",
-		framework.NewController(context.Background(), &resolver{}),
+	ctx := filteredinformerfactory.WithSelectors(context.Background(), v1alpha1.ManagedByLabelKey)
```

Comment:
I don't know why this didn't barf with `Unable to fetch labelkey from context` previously, but it started doing that in this PR until I added this and changed the next lines here and in the other `Resolver` `main.go`s.